prov/sockets/src/sock_cntr.c

```diff
@@ -420,7 +421,11 @@ static int sock_cntr_verify_attr(struct fi_cntr_attr *attr)
 	switch (attr->wait_obj) {
 	case FI_WAIT_NONE:
 	case FI_WAIT_UNSPEC:
+#ifndef _WIN32
 	case FI_WAIT_MUTEX_COND:
+#else /* _WIN32 */
+	case FI_WAIT_CRITSEC_COND:
+#endif /* _WIN32 */
 	case FI_WAIT_SET:
 	case FI_WAIT_FD:
```

Comment:
> FI_WAIT_FD may be used on windows - there is WSAPoll function which may
> wait for sockets, its't it?

Windows does not have fd's in the same sense as Linux.  Linux defines an fd as an int, and it can be a file or socket.  WSAPoll takes a SOCKET only, not a file, and is different from Linux in this matter.

I'm not sure about how to handle this.  Letting FI_WAIT_FD work on windows as a SOCKET seems to make sense, but from the perspective of an application using libfabric on both windows and linux, the semantics it can rely on aren't the same.